postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -295,51 +296,52 @@ def _collect_statement_samples(self):
 
     def _can_explain_statement(self, obfuscated_statement):
         if obfuscated_statement.startswith('SELECT {}'.format(self._explain_function)):
-            return False
+            return False, DBExplainError.no_plans_possible
```

Comment:
Just realized since `_can_explain_statement` always returns the same `DBExplainError.no_plans_possible` in case of error it would be simpler to avoid changing this function at all and just have the caller use `DBExplainError.no_plans_possible` itself when the result here is `False`

Reply:
Ah, totally right, nice catch 👍